client/src/components/Header/Header.scss

```diff
@@ -1,37 +1,86 @@
+@import "../../../node_modules/bootstrap/scss/functions";
+@import "../../../node_modules/bootstrap/scss/variables";
+@import "../../../node_modules/bootstrap/scss/mixins";
+
 .header {
     display: flex;
-    padding: 2rem;
     font-size: 0.875rem;
     position: relative;
+    padding: 1rem;
+    @include media-breakpoint-up(lg) {
+        padding: 2rem;
+    }
 
-    .left {
-        flex: 20%;
+    .header-icon {
+        object-fit: cover;
     }
 
-    .center {
-        flex: 60%;
+    .desktop-links {
+        display: none;
+        @include media-breakpoint-up(lg) {
+            flex: 20%;
+            display: inline-block;
+            margin-left: 2rem;
```

Comment:
There were a lot of places where percents were being used for things like padding and margin, please use rem for that instead https://webdesign.tutsplus.com/tutorials/why-you-should-be-using-rem-based-layouts--cms-27828